specs/deneb/p2p-interface.md

```diff
@@ -203,6 +203,15 @@ Per `context = compute_fork_digest(fork_version, genesis_validators_root)`:
 |--------------------------------|---------------------|
 | `DENEB_FORK_VERSION` and later | `deneb.BlobSidecar` |
 
+###### Blob retrieval via local execution layer client
+
+In addition to `BlobSidecarsByRoot` requests, recent blobs MAY be retrieved by querying the Execution Layer (i.e. via `engine_getBlobsV1`).
+Honest nodes SHOULD query `engine_getBlobsV1` as soon as they receive a valid gossip block that contains data, and import the returned blobs.
+
+When clients use the local execution layer to retrieve blobs, they MUST behave as if the corresponding `blob_sidecar` had been received via gossip. In particular they MUST:
+* publish the corresponding `blob_sidecar` on the `blob_sidecar_{subnet_id}` subnet.
```

Comment:
Needs a blank line before bullet points now. Otherwise it will not render correctly in mkdocs. See:
* https://github.com/ethereum/consensus-specs/issues/4136
```suggestion
When clients use the local execution layer to retrieve blobs, they MUST behave as if the corresponding `blob_sidecar` had been received via gossip. In particular they MUST:

* publish the corresponding `blob_sidecar` on the `blob_sidecar_{subnet_id}` subnet.
```